libs/designer-ui/src/lib/card/card.less

```diff
@@ -1072,6 +1080,14 @@
         color: @ms-color-black;
       }
     }
+
+    &--no-action {
+      border: 1px dashed @ms-color-primaryBorder;
+
+      &:hover {
```

Comment:
We don't need a hover effect since this is an un-interactive component.  If this is inheriting a bunch of styles lets just remove all the intherited classes from the component and have a fresh class, there is only going to be a few styles on this component anyways.